configurations/manager/ubuntu22.yaml

```diff
@@ -1,2 +1,2 @@
-ami_id_monitor: 'ami-09db26f1ef0a9f406'  # Canonical, Ubuntu, 22.04 LTS, amd64 jammy image build on 2022-05-06
+ami_id_monitor: 'scylladb-monitor-4-6-2-2024-02-13t08-06-04z'
```

Comment:
It's a bit problematic

Jobs using this config would stick with it, 

And won't ever get an updated monitor image, and at some point you'll be missing some monitoring features

Reply:
Why it was set as default in `defaults/aws_config.yaml` in such case?

Reply:
And should I then just leave `ami-09db26f1ef0a9f406` image without changes?

Reply:
The idea is that there are specific places where we set the default image per backend

If we clone it to more places no one would update them when there's a new version of it

So I would recommend removing it from here, but keep in mind it might get updated with a different version at some point with you knowing about it

Reply:
ah, okay, I got your point. I misunderstood what you meant above. My idea was to specify it explicitly to make visible without going deeper into additional config files. But I agree it can lead to sticking with this image for a long time. Will update the PR.

Reply:
Fixed